pyeudiw/jwt/jws_helper.py

```diff
@@ -134,14 +131,19 @@ def sign(
             signing_key.pop("kid", None)
 
         signer = JWS(payload, alg=signing_alg)
-        
         if serialization_format == "compact":
             try:
-                signed = signer.sign_compact([key_from_jwk_dict(signing_key)], protected=protected, **kwargs)
+                signed = signer.sign_compact(
+                    [key_from_jwk_dict(signing_key)], protected=protected, **kwargs
+                )
                 return signed
             except Exception as e:
                 raise JWSSigningError("Signing error: error in step", e)
-        return signer.sign_json(keys=[key_from_jwk_dict(signing_key)], headers=[(protected, unprotected)], flatten=True)
+        return signer.sign_json(
+            keys=[key_from_jwk_dict(signing_key)],
```

Comment:
don't repeat this twice `key_from_jwk_dict(signing_key)`

please assign the result to a variable and reuse it